src/modules/client/pages/ni/customers/CustomersDirectory.vue

```diff
@@ -194,22 +194,22 @@ export default {
     async getCustomers () {
       try {
         this.tableLoading = true;
-        this.customers = await Customers.list();
+        return Customers.list();
       } catch (e) {
-        this.customers = [];
         console.error(e);
       } finally {
         this.tableLoading = false;
       }
     },
     async getCustomersFirstIntervention () {
       try {
-        this.firstInterventions = await Customers.listWithFirstIntervention();
-        this.customers = this.customers.map(customer => ({
+        this.firstInterventions = Object.freeze(await Customers.listWithFirstIntervention());
+        const customers = await this.getCustomers();
+        this.customers = Object.freeze(customers.map(customer => ({
```

Comment:
Le loading du tableau ne fonctionne plus bien. Je pense que c'est parce que this.customers est defini ici, donc le loading est deja remis a false.